web/main/views.py

```diff
@@ -2132,6 +2132,7 @@ def get(self, request, casebook, section):
             {
                 "casebook": casebook,
                 "section": section,
+                "ali_license": section.get_ali_license_text(),
```

Comment:
These want to use section.ali_license instead of the method, to avoid recalculating, right?